src/phoenix/server/api/types/Project.py

```diff
@@ -536,6 +540,97 @@ async def validate_span_filter_condition(self, condition: str) -> ValidationResu
                 error_message=e.msg,
             )
 
+    @strawberry.field(
+        description="Hourly span count for the project.",
```

Comment:
why not make the bucketing configurable like the other charts. I think we might want to be able to adjust?